sycl/doc/GetStartedGuide.md

```diff
@@ -5,23 +5,34 @@ and a wide range of compute accelerators such as GPU and FPGA.
 
 ## Table of contents
 
-* [Prerequisites](#prerequisites)
-  * [Create DPC++ workspace](#create-dpc-workspace)
-* [Build DPC++ toolchain](#build-dpc-toolchain)
-  * [Build DPC++ toolchain with libc++ library](#build-dpc-toolchain-with-libc-library)
-  * [Build DPC++ toolchain with support for NVIDIA CUDA](#build-dpc-toolchain-with-support-for-nvidia-cuda)
-  * [Build Doxygen documentation](#build-doxygen-documentation)
-* [Use DPC++ toolchain](#use-dpc-toolchain)
-  * [Install low level runtime](#install-low-level-runtime)
-  * [Obtain prerequisites for ahead of time (AOT) compilation](#obtain-prerequisites-for-ahead-of-time-aot-compilation)
-  * [Test DPC++ toolchain](#test-dpc-toolchain)
-  * [Run simple DPC++ application](#run-simple-dpc-application)
-  * [Code the program for a specific GPU](#code-the-program-for-a-specific-gpu)
-  * [Using the DPC++ toolchain on CUDA platforms](#using-the-dpc-toolchain-on-cuda-platforms)
-* [C++ standard](#c-standard)
-* [Known Issues and Limitations](#known-issues-and-limitations)
-* [CUDA back-end limitations](#cuda-back-end-limitations)
-* [Find More](#find-more)
+- [Getting Started with oneAPI DPC++](#getting-started-with-oneapi-dpc)
+  - [Table of contents](#table-of-contents)
```

Comment:
Are these links really needed?

I see a lot of changes to this section, which are not related to enabling AMD GPU support. Could you move them to a separate PR, please?

I expect only two new links to the sections added for AMD GPU.